### ISSUE TITLE
Wizard styles to be based on hidden attribute instead of aria-hidden

### ISSUE THREAD

**Issue by sosharma1403** [vmware/clarity#2399]
**Select one ...** (check one with "x")

```
[ ] bug
[ ] feature request
[X] enhancement
```

### Expected behavior
Wizard styles should be based on `hidden` attribute instead of `aria-hidden`

### Actual behavior
Wizard styles are based on `hidden` attribute.

### Reproduction of behavior
https://vmware.github.io/clarity/documentation/v0.12/wizards#examples

### Environment details

* **Angular version:** 6.x.x

* **Clarity version:**: 0.12.1

* **OS and version:**: Win 7

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by hippee-lee** [comment]
Hi @sosharma1403 - do you have any more information to provide so we can understand the enhancement you are proposing?
